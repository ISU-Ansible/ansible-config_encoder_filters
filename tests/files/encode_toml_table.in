tab1:
  text: some text
  arr11:
    - 123
    - 987.654
  bool1: yes
  bool2: "false"
