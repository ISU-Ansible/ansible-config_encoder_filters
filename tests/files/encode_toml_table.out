[tab1]
arr11 = [123, 987.654]
bool1 = true
bool2 = "false"
text = "some text"
